okapi/services/caches/reports/status/docs.xml

```diff
@@ -0,0 +1,40 @@
+<xml>
+    <brief>Query the status of geocache reports</brief>
+    <issue-id>TODO</issue-id>
+    <desc>
+        <p>This method queries the status of one or more geocache reports that were
+        submitted trough <a href="submit.html">services/caches/reports/submit</a>.</p>
```

Comment:
I think there was a plugin for generating values for hrefs (which takes the full path of the method and returns a valid URL). It's better for future-compatibility.

Reply:
ok, it's %OKAPI:methodargref